recipes/xkbcommon/all/conanfile.py

```diff
@@ -73,5 +73,12 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.components['libxkbcommon'].libs = ['xkbcommon']
+        self.cpp_info.components['libxkbcommon'].name = 'xkbcommon'
+        self.cpp_info.components['libxkbcommon'].requires = ['xorg::xorg']
+        if self.options.with_x11:
+            self.cpp_info.components['libxkbcommon-x11'].libs = ['xkbcommon-x11']
+            self.cpp_info.components['libxkbcommon-x11'].name = 'xkbcommon-x11'
+            self.cpp_info.components['libxkbcommon-x11'].requires = ['libxkbcommon']
+            self.cpp_info.components['libxkbcommon-x11'].requires.append('xorg::xorg')
```

Comment:
```suggestion
```

This line shouldn't be needed, it is a transitive dependency via component `libxkbcommon`

Reply:
`libxkbcommon` and `libxkbcommon-x11` do not use the same xorg components. The end goal is to do ` self.cpp_info.components['libxkbcommon'].requires.append('xorg::xkeyboard-config')` and ` self.cpp_info.components['libxkbcommon-x11'].requires.extend(['xorg::xcb', xorg::xcb-xkb'])` when https://github.com/conan-io/conan-center-index/pull/2362 is merged

Reply:
Merged! 😉 

Maybe we can update this PR accordingly, thanks!